example/Benchmark_spiner.cpp

```diff
@@ -0,0 +1,380 @@
+/*
+  This script compares the performance of EOSPAC SpinerEOSDependsRhoT and SpinerEOSDependsRhoSie in the following cases:
+
+  1. P(rho, T) 
+  2. e(rho, T) 
+  3. P(rho, e(rho, T)) 
+  4. T(rho, e(rho, T)) 
+
+  The output of this script is as follows:
+  1. For each material, model, and lookup type, a .csv of the grid values (to compare accuracy)
+  2. A .csv of the lookup times for x-amount of trials (to compare performance)
+
+  Example usage: ./Benchmark ./output_dir 100 100 ./tables/my_eos.sp5 2020 3720
+
+  Authors: Erin O'Neil and Joshua Basabe
+ */
+
+//C++ Headers
+#include <algorithm>
+#include <chrono>
+#include <cstdlib>
+#include <cstring>
+#include <iostream>
+#include <limits>
+#include <string>
+#include <vector>
+#include <fstream>   
+#include <iomanip> 
+
+
+//Data headers
+#include <hdf5.h>
+#include <hdf5_hl.h>
+
+//Spiner headers
+#include <ports-of-call/portability.hpp>
+#include <singularity-eos/base/sp5/singularity_eos_sp5.hpp>
+#include <spiner/databox.hpp>
+#include <spiner/interpolation.hpp>
+#include <spiner/sp5.hpp>
+#include <spiner/spiner_types.hpp>
+#include <ports-of-call/portability.hpp>
+
+//Get EOS Models
+#include <singularity-eos/eos/eos.hpp>
+#include "singularity-eos/eos/eos_spiner.hpp"
+#include <eospac-wrapper/eospac_wrapper.hpp>
+
+using namespace singularity;
+using namespace EospacWrapper;
+using namespace std::chrono;
+#include <experimental/filesystem> //not sure why I had to do this
+namespace fs = std::experimental::filesystem;
```

Comment:
I think this might be a standards thing. This might go away in the future. @mauneyc-LANL ?